package.json

```diff
@@ -18,6 +18,6 @@
     "@duckduckgo/autoconsent": "github:duckduckgo/autoconsent#v3.0.0",
     "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.6.0",
     "@duckduckgo/privacy-reference-tests": "github:duckduckgo/privacy-reference-tests#1.0.0",
-    "@duckduckgo/privacy_dashboard": "github:more-duckduckgo-org/duckduckgo-privacy-dashboard#ca7b5f444455b587a6bafab9c2b8a44ec99dad53"
+    "@duckduckgo/privacy_dashboard": "github:duckduckgo/privacy-dashboard#main"
```

Comment:
nit: should be pointing to a version rather than main to avoid future errors. Suggest to push the frontend team to do proper releases :)

Reply:
Yes, totally agree, forgot to mention that they are already aware of this and we will have proper release numbers.

I'm going to use hash commit until releases are ready. I was using `main` during development just for convenience, but once this is released, we would like to target something "static".